CalPatRec/src/AgnosticHelixFinder_module.cc

```diff
@@ -970,9 +980,10 @@ namespace mu2e {
 
     // seed point
     for (size_t i = 0; i < _tcHits.size() - _minLineSegmentHits; i++) {
-      if (_tcHits[i].inHelix == true || _tcHits[i].hitIndice < 0) { continue; }
-      if (_tcHits[i].used == false) { continue; }
-      if (_tcHits[i].notOnSegment == false) { continue; }
+      if (_tcHits[i].inHelix == true || _tcHits[i].used == false
```

Comment:
There's no need to compare a boolean to a value: 
_tcHits[i].inHelix || (!_tcHits[i].used)
will evaluate the same